src/Dhall.hs

```diff
@@ -203,9 +212,32 @@ inputFromWith
     -- ^ The Dhall program
     -> IO a
     -- ^ The decoded value in Haskell
-inputFromWith filename (Type {..}) ctx n txt = do
+inputFromWith filename ty ctx n txt =
+  inputDirFromWith "." filename ty ctx n txt
+
+{-| Extend 'inputFrom' with a root directory to resolve imports relative
+    to, a file to mention in errors as the source, a custom typing
+    context, and a custom normalization process.
+
+@since 1.6
```

Comment:
Did you mean `@since 1.15` or `@since 1.16`?

Reply:
Oops, yes. Might be contagious!